Sources/Rownd/framework/Constants.swift

```diff
@@ -21,6 +21,27 @@ func getFrameworkVersion() -> String {
     return bundleVersion
 }
 
+struct FrameworkFeature: Codable {
+    var name: String
+    var enabled: String
+}
+
+func getFrameowrkFeatures() -> String {
+    var frameworkFeatures: [FrameworkFeature] = [FrameworkFeature(name: "openEmailInbox", enabled: "true")]
+    
+    let encoder = JSONEncoder()
+    encoder.dataEncodingStrategy = .base64
+    
+    do {
+        let encodedData = try encoder.encode(frameworkFeatures)
+        return String(data: encodedData, encoding: .utf8) ?? "{}"
+    } catch {
+        print("ROWND: Couldn't encode Framework Features")
```

Comment:
You can use something like: `logger.warn("Failed to encode framework features: \(String(describing: error)))")` here to provide better formatting and error details (although it theoretically won't ever fail.

Reply:
Updated. We'll need to merge the Hub PR before this gets merged. The JS fails without the rownd.setSesssionStorage api